release/community-stable/oraclelinux/docker/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN \
 	    # required for help in powershell
 	    less \
       # install EPEL repository
-	    epel-release \
+	    oracle-epel-release-el7 \
```

Comment:
`epel-release` seems to no longer be accurate:
![image](https://user-images.githubusercontent.com/1616060/70418228-a6f04680-1a28-11ea-8da4-dba966d76b23.png)

https://docs.oracle.com/en/operating-systems/oracle-linux/notice-yum-modularization/alternate.html

After changing this line, I'm able to locally run `(Get-ChildItem /usr/lib*/*/gssntlmssp.so -Recurse |select-object -first 1).FullName` and have it return a result just fine. I'm not sure why it's not working in the PR build.

Reply:
Why does this help?

Reply:
@RDIL - It doesn't do anything for image size, but, as (poorly) stated, it _should_ be fixing the `oraclelinux` build since `yum install epel-release` appears to no longer work.

Reply:
The test _appears_ to be failing when it shouldn't, but that's probably just me missing something obvious.

Reply:
It looks like this fixes the build, however now the test for `gss-ntlmssp` is failing.

Reply:
Yeah, it can't find `gss-ntlmssp` on the image.

Reply:
Right. I'm saying I ran the PS command on a container I built locally after updating the epel package name, and it seemed to find `gss-ntlmssp` just fine.

Reply:
Can you open the image's `meta.json` and set `SkipGssNtlmSspTests` to `true`?  That will make the build pass for now.